schemes/check/ccpp_prebuild_config.py

```diff
@@ -45,6 +45,9 @@
 # Directory where to put all auto-generated physics caps
 CAPS_DIR = '.'
 
+# Directory where the suite definition files are stored
+SUITES_DIR = '../../../../../src/tests'
```

Comment:
Not very pythonic. Any reason you are not using `os.path`?

Reply:
I guess no, no particular reason. Would have to redo the entire file, though, since all other entries are in the same way. I will probably leave this for another cleanup PR, if it gets done at all before moving to cap_gen. Note that I had to change a few more files, including some changes that need to be tested with NEMSfv3gfs. I will conduct the tests in conjunction with upcoming PRs (will be listed below shortly) and have them merged together.